chore: add FAQ about nextjs CSS issue (#5)
moxystudio/react-lib-template: README.md

*Note: Everytime a change is made to the package a rebuild is required to reflect those changes on the demo.*

## FAQ

### I can't override the component's CSS, what's happening?

> There is an ongoing [next.js issue](https://github.com/zeit/next.js/issues/10148) about the loading order of modules and global CSS in development mode. This has been fixed in [v9.3.6-canary.0](https://github.com/zeit/next.js/releases/tag/v9.3.6-canary.0), so you can either update `next.js` to a version higher than `v9.3.5`, or simply increase the CSS specificity when overriding component's classes, as we did in the [`demo`](./demo/pages/index.module.css), e.g. having the page or section CSS wrap the component's one.

Reviewer: I would change this:
> e.g. having the page or section CSS wrap the component's one.

to this:
> e.g. giving a class to the component's wrapper element.

Author: I get what you're saying but that can be misleading, as it's not enough to give a class to the wrapper element in the HTML, you need to have a hierarchy in the CSS file. 

Both state more or less the same, but I think mine is more explicit, although still not perfect.

Reviewer: No strong opinion here. I'm good with it as is 👍

> There is currently [an issue](https://github.com/zeit/next.js/issues/10148) on nextjs with the loading order of modules and global CSS in development mode. This has been fixed in [v9.3.6-canary.0](https://github.com/zeit/next.js/releases/tag/v9.3.6-canary.0), so you can either update `nextjs` to a version higher than `v9.3.5`, or simply increase the CSS specificity when overriding the component's class, as we did in the [`demo`](./demo/pages/index.module.css), e.g. with having the page or section CSS wrap the component's one.

Reviewer 2: ```suggestion
There is an ongoing [issue](https://github.com/zeit/next.js/issues/10148) with next.js about the loading order of modules and global CSS in development mode. This has been fixed in [v9.3.6-canary.0](https://github.com/zeit/next.js/releases/tag/v9.3.6-canary.0), so you can either update `nextjs` to a version higher than `v9.3.5`, or simply increase the CSS specificity when overriding component's classes, as we did in the [`demo`](./demo/pages/index.module.css), e.g. having the page or section CSS wrap the component's one.
```
